rllib/env/multi_agent_env_runner.py

```diff
@@ -361,6 +363,9 @@ def _sample(
             # Try stepping the environment.
             results = self._try_env_step(actions_for_env)
             if results == ENV_STEP_FAILURE:
+                logging.warning(
```

Comment:
Nice message. Can we unify all messages patterns?

Reply:
I didn't notice single pattern established across RLlib. If you like this direction I can unify for this PR first, then expand and unify component by component. WDYT?

Reply:
Yea, sounds great. Following up in a new another PR.